### ISSUE TITLE
Change Request: restrict linting per file type in CLI

### ISSUE THREAD

**Issue by tudobeleza** [eslint/eslint#19246]
### ESLint version

9.17.0

### What problem do you want to solve?

I would like to have a CLI option to restrict the linting process of a script to a subset of file types.

For example, in package.json, currently we do:

```
"scripts": {
    "_linter:analyze": "eslint ./",
```

That script analyzes .js files, .css files, .md files and .json files, all together, if you have @eslint/css, @eslint/markdown, @eslint/json and eslint installed.
This is nice most of the times, but sometimes, you need to restrict per file type... and you want to restrict in the CLI. So it would be convenient if we could do:

```
"scripts": {
    "_css:analyze": "eslint ./ --language:css",
    "_js:analyze": "eslint ./ --language:js",
    "_json:analyze": "eslint ./ --language:json",
    "_md:analyze": "eslint ./ --language:markdown",
```

Can you please add this `--language` feature to CLI?

### What do you think is the correct solution?

Provided above.

### Participation

- [ ] I am willing to submit a pull request for this change.

**Comment by aladdin-add** [comment]
This is already possible:

```js
"scripts": {
    "_css:analyze": "eslint ./**/*.css --cache",
```

I didn't see the value to adding a new option. thoughts? @eslint/eslint-team

**Comment by nzakas** [comment]
You can always pass whatever files you want to limit the ESLint run to on the command line. 

No work to do here, so closing.
